### PR TITLE
Support for Ruby 2.7

### PR DESCRIPTION
**Pull request by 0exp**
dinosaur project upgrade requirement :)

---

- CI: + ruby 2.7
- Version: bump to 0.4.0
- Gem: Support for ruby 2.7

### CODE REVIEW

File: lib/hourly_logger_rotator/patch/ruby_2_4.rb [hunk 1]
```diff
@@ -5,7 +5,13 @@ module Patch
     module Ruby_2_4 # rubocop:disable Naming/ClassAndModuleCamelCase
       SiD = Logger::Period::SiD
 
-      def initialize(log = nil, shift_age: nil, shift_size: nil, shift_period_suffix: nil)
+      def initialize(
+        log = nil,
+        shift_age: nil,
+        shift_size: nil,
+        shift_period_suffix: nil,
+        binmode: nil
```
Comment (0exp) [inline lib/hourly_logger_rotator/patch/ruby_2_4.rb:14]: dunno why `binmode` is needed 🤔

**Comment by tycooon** [comment]
Didn't we swtich to logrotate? 🤔

**Comment by 0exp** [comment]
Dinosaur - dunno 🤔

**Comment by 0exp** [comment]
> Didn't we swtich to logrotate? 🤔

Dinosaur? Dunno 🤔

File: lib/hourly_logger_rotator/patch/ruby_2_4.rb (hunk 1 above)
Comment (tycooon) [reply lib/hourly_logger_rotator/patch/ruby_2_4.rb:14]: 🤔

**Review by tycooon — APPROVED** [review]

**Pull request merged**